packages/bentley/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@bentley/grouping-mapping-widget",
+  "version": "0.0.15",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/bentley/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/main.bundle.js",
+  "types": "lib/grouping-mapping-widget.d.ts",
+  "scripts": {
+    "build": "webpack && npm run copy-files ",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "copy-files": "ncp README.md lib/README.md && ncp LICENSE.md lib/LICENSE.md",
```

Comment:
Why are you copying these to the `lib/` folder? Having them at the root should be enough for them to show on npmjs.

Reply:
I see. Copied them so that they show up in node_modules in the downloaded package, but if it is not needed happy to remove.

Reply:
run `npx pkgfiles` to see what gets delivered in your pkg